packages/react-ui-ag/src/Modals/__tests__/MoreOptionsModal-test.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import ThemedMoreOptionsModal from '../MoreOptionsModal'
+import theme from './mocks/theme'
+
+const MoreOptionsModal = ThemedMoreOptionsModal.WrappedComponent
+
+describe('MoreOptionsModal', () => {
+  const props = {
+    theme,
+    closeModal: jest.fn(),
+    filterTotal: 0,
+    total: 0,
+    isOpen: true,
+    onSubmit: jest.fn(),
+    clearFilters: jest.fn(),
+    restoreFilters: jest.fn(),
+  }
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it('has header', () => {
+    const wrapper = mount(<MoreOptionsModal {...props} />)
```

Comment:
Please use `shallow` where possible.

Reply:
ok will do

Reply:
I tried changing it, but due to it being themed. When comparing text like for the header,  shallow is not deep enough. Instead of returning the text it returns `<Themed(Text) />` .

Reply:
@tadjohnston Yea I am doing that at the top of the file, but it is still returning `<Themed(Text) />` instead of just the text